app/src/main/res/layout/fragment_new_settings.xml

```diff
@@ -36,8 +36,24 @@
             android:background="@drawable/settings_item_selector"
             android:focusable="true"
             android:layout_gravity="center_vertical">
+
             <requestFocus/> <!-- Otherwise focus gets stuck on outer id/container. -->
+
+            <!-- a11y: FRAGILE! When the parent layout is focused, we want to speak, "Checkbox (un)checked", based
+                 on the state of the checkbox child. However, unlike for TextViews, VoiceView does not read the
+                 checked state of child CheckBoxes so we have to work around this:
+
+                 When VoiceView is enabled, we dynamically change focusability so the parent layout is
+                 unfocusable and we focus the CheckBox instead, which announces the correct data. Since the
```

Comment:
Okay, it looks like this hack is "when accessibility is turned on, we sacrifice correct visual highlighting in order to get the correct content description". Can you make that clear by putting this goal as the first sentence of this paragraph?